tests/framework/utils/k8s_helper.go

```diff
@@ -64,6 +63,29 @@ const (
 	hostnameTestPrefix = "test-prefix-this-is-a-very-long-hostname-"
 )
 
+func TestPlatform() string {
+	return GetEnvVarWithDefault("TEST_PLATFORM", "openshift")
```

Comment:
how about blank by default? We don't want to default to openshift

Reply:
by default making it k8s according to [this](https://github.com/rook/rook/pull/6185#discussion_r485190609) missed earlier.